### ISSUE TITLE
Selected store not preserved after forced logout

### ISSUE THREAD

**Issue by CarlosNZ** [openmsupply/open-msupply#1828]
I'm not 100% sure of the conditions to recreate this, I'll look into a bit more thoroughly later today.

But basically, if there is some kind of authorisation error (in my case it was when the server had restarted, so changed the JWT private key, presumably), you get logged out. But when you log back in, the selected store is not preserved. I can see that the selected store was preserved in local storage right up until login.

When you log out manually (or I think after a timeout too), when re-logging in, the selected store is preserved. 

So there must be something slightly different going on in these two cases -- one is using the local storage to get the preferred store, and the other is getting it from the server.

Maybe there's a reason why this is intentional behaviour?

**Comment by mark-prins** [comment]
Retested with current develop branch and cannot repro.
Would it be ok to close the issue @CarlosNZ?

**Comment by CarlosNZ** [comment]
> Would it be ok to close the issue @CarlosNZ?

Yeah, probably. I'll have a go at reproducing it tomorrow morn and if I can't, I'll close the issue.

**Comment by CarlosNZ** [comment]
Okay, yeah, doesn't seem to be a problem any more. Will close 👍
